ui/app/store/actions.js

```diff
@@ -432,22 +432,24 @@ export function unlockHardwareWalletAccount(index, deviceName, hdPath) {
   return (dispatch) => {
     dispatch(showLoadingIndication());
     return new Promise((resolve, reject) => {
-      background.unlockHardwareWalletAccount(
-        index,
-        deviceName,
-        hdPath,
-        (err) => {
-          dispatch(hideLoadingIndication());
-          if (err) {
-            log.error(err);
-            dispatch(displayWarning(err.message));
-            reject(err);
-            return;
-          }
+      setTimeout(() => {
```

Comment:
This obviously needs to be removed.  We need to send unlock requests one at a time to avoid timing conflicts with the data returned.